cmd/kubectl_token.go

```diff
@@ -146,6 +156,27 @@ func runCredential(ctx *cli.Context) error {
 	return json.NewEncoder(os.Stdout).Encode(newCred)
 }
 
+func deleteCachedCredential(ctx *cli.Context) error {
+	dir, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+	cacheDir := filepath.Join(dir, kubeConfigCache)
+	if len(ctx.Args()) == 0 {
```

Comment:
Removing all tokens when not specifying a arg seems heavy. We either need to document this in the param command or force them to set an arg like `all` to get this behavior.

Reply:
Yep 💯. I moved it to a subcommand, added `all` to remove all tokens and usage for it